restaurants/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+from django.http        import JsonResponse
+from django.views       import View
+from django.db.models   import Avg
 
-# Create your views here.
+from restaurants.models import Restaurant
+
+class RestaurantDetailView(View):
+    @ConfirmUser
+    def get(self, request, restaurant_id):
+        try:
+            restaurant = Restaurant.objects.get(id=restaurant_id)
+            is_wished  = request.user.wishlist_restaurants.filter(id=restaurant_id).exists() if request.user else False
+
+            reviews                   = restaurant.review_set.all()
+            average_rating            = reviews.aggregate(Avg("rating"))["rating__avg"]
+            review_count              = {
+                "total"        : reviews.count(),
+                "rating_one"   : reviews.filter(rating=1).count(),
+                "rating_two"   : reviews.filter(rating=2).count(),
+                "rating_three" : reviews.filter(rating=3).count(),
+                "rating_four"  : reviews.filter(rating=4).count(),
+                "rating_five"  : reviews.filter(rating=5).count(),
+            }
+
+            result = {
+            "id"             : restaurant.id,
+            "sub_category"   : restaurant.sub_category.name,
+            "name"           : restaurant.name,
+            "address"        : restaurant.address,
+            "phone_number"   : restaurant.phone_number,
+            "coordinate"     : restaurant.coordinate,
+            "open_time"      : restaurant.open_time,
+            "updated_at"     : restaurant.updated_at,
+            "is_wished"      : is_wished,
+            "review_count"   : review_count,
+            "average_rating" : average_rating,
```

Comment:
dictionary 내부에 indentation만 맞춰주시면 좋을 것 같습니다!
```suggestion
                "id"             : restaurant.id,
                "sub_category"   : restaurant.sub_category.name,
            "name"           : restaurant.name,
            "address"        : restaurant.address,
            "phone_number"   : restaurant.phone_number,
            "coordinate"     : restaurant.coordinate,
            "open_time"      : restaurant.open_time,
            "updated_at"     : restaurant.updated_at,
            "is_wished"      : is_wished,
            "review_count"   : review_count,
            "average_rating" : average_rating,
```